libs/components/help-inline/src/lib/modules/help-inline/help-inline.component.ts

```diff
@@ -43,7 +52,7 @@ export class SkyHelpInlineComponent {
   protected popoverTemplate: TemplateRef<unknown> | undefined;
   protected isPopoverOpened: boolean | undefined;
 
-  #_popoverContent: string | TemplateRef<unknown> | undefined;
+  #helpSvc = inject(SkyHelpService);
```

Comment:
Fixed.

Reply:
Also, putting injected services is something I've started doing in other places as well since it flows well in cases where another property uses the injected service to instantiate its value, for example:

```
#idSvc = inject(SkyIdService);

protected controlId = this.#idSvc.generateId();
```